.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,193 @@
+
+### Clone and install repository 
+- how to clone and get going with the kirby repo
+
+### Node 
+- Which version should people use, and perhaps a recommendation on which tool people can use to manage versions. 
+
+### Tslint 
+- People should make sure to have it enabled to avoid unnecessary comments during code review
+
+### Prettier
+- It is a good idea to use format-on-save
+
+## Creating pull-requests 
+<!-- This section is responsible for guiding contributors on how they should submit pull-requests in a way that ensures the most swift process and teach them how to catch low-hanging issues there might come up in code-review. -->
+
+### Before you submit  
+- Make sure you have completed the steps in: [Setup tooling for contribution](#setup-tooling-for-contribution)
+- Make sure you have written tests according to the guidelines described in [The good: test](#test)
+- Do a self-review <!-- We could link to self-review in the templates and have people check it off? -->
+
+### After you submit 
+- Post in Kirby-guild 
+- Code reviews are meant to align your code with existing code 
+  - While your implementation might be perfectly valid, we are aiming for uniformity across the codebase. 
+- Please don't fire & forget, i.e. commit & run - make sure to stay responsive and come back to answer comments on your code review :)
+## The good:
+<!-- The following sections have the responsibility of explaining contributors our best pratices such that they can more easily have their changes implemented in kirby --> 
+<!-- Ideally we should be able to refer to these sections if people break the guidelines in their issue or pull-requests, to make sure we are all preaching the same guidelines -->
+
+### Issue 
+- General properties we are looking for and how to achieve them 
+  - Readability 
+  - Easily understandable 
+    - No internal lingo 
+  - Proof-read
+  - Self-contained
+    - Screenshots if necessary
+    <!-- I think the following point is important if Kirby is to be spread out amongst several teams -->
+    - Don't assume people have knowledge outside of Kirby - contributors might come from many different projects 
+
+### Commit
+- Focus on one thing per commit 
+- Keep changes as small as possible 
+- Only commit related functinoality for each commit 
+- Commit often
+- Strive for each commit to compile/build and pass tests 
+- Use [gitmoji](https://gitmoji.dev) when possible - point to VSCode extension and CLI tool
+- sidenote: when merging the CI pipeline combines each commit to a merge commit containing all commit messages
+
+### Self-review 
+- Don't change files that is not related to your pull-request
+  - for example don't reorder imports in files that is not directly related to the issue at hand
+- Check for unused imports
+- Other self review points go here
+
+### Pull-request
+- How is a pull-request that is easily reviewed constructed?
+
+### Test
+- How should testing be done 
+
+## I have a question related to Kirby
```

Comment:
This section should probably be moved to a SUPPORT.md file (https://github.com/kirbydesign/designsystem/issues/1462)